src/PythonProjectBootstrapper/package/hooks/post_gen_project.py

```diff
@@ -186,106 +186,128 @@ def SavePrompts() -> None:
         """,
     )
 
-    prompts["Update README.md"] = textwrap.dedent(
+{% if cookiecutter.openssf_best_practices_badge_id != "none" %}

```

Comment:
Nit: same as earlier with the strip and lower case

Reply:
Good catch. I will add this to a separate commit after the other changes go in.